tensorflow/core/kernels/mkl/mkl_quantize_op.cc

```diff
@@ -88,21 +107,30 @@ class MklReorderWithScalePrimitive : public MklPrimitive {
   std::shared_ptr<primitive> GetPrimitive() { return context_.reorder_prim; }
 
   void Execute(void* src_data, void* dst_data,
+#ifdef ENABLE_ONEDNN_V3
+               void* scale_data,
+#endif  // ENABLE_ONEDNN_V3
                std::shared_ptr<stream> reorder_stream) {
 #ifdef DNNL_AARCH64_USE_ACL
     mutex_lock lock(primitive_execution_mu_);
 #endif
-#ifndef ENABLE_ONEDNN_OPENMP
+#if !defined(ENABLE_ONEDNN_OPENMP) && !defined(ENABLE_ONEDNN_V3)
     context_.src_mem->set_data_handle(src_data, *reorder_stream);
     context_.dst_mem->set_data_handle(dst_data, *reorder_stream);
 #else
     context_.src_mem->set_data_handle(src_data);
     context_.dst_mem->set_data_handle(dst_data);
-#endif  // !ENABLE_ONEDNN_OPENMP
+#ifdef ENABLE_ONEDNN_V3
+    context_.scale_mem->set_data_handle(scale_data);
+#endif  // ENABLE_ONEDNN_V3
+#endif  // !ENABLE_ONEDNN_OPENMP && !ENABLE_ONEDNN_V3
```

Comment:
Should the `#ifdef ENABLE_ONEDNN_V3` be after `#endif // !ENABLE_ONEDNN_OPENMP && !ENABLE_ONEDNN_V3` instead? Otherwise the code in this block would be unreachable (`ENABLE_ONEDNN_V3` must be false to enter the outer if).
```suggestion
#endif  // !ENABLE_ONEDNN_OPENMP && !ENABLE_ONEDNN_V3
#ifdef ENABLE_ONEDNN_V3
    context_.scale_mem->set_data_handle(scale_data);
#endif  // ENABLE_ONEDNN_V3
```

Reply:
Fixed.